grails-app/utils/au/org/emii/portal/HtmlCharacterEntityReferences.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
```

Comment:
I found this, discussing apache and GPL3 http://www.apache.org/licenses/GPL-compatibility.html . I don't see there's a problem.